src/constants.js

```diff
@@ -25,6 +25,7 @@ export function generatePrefixes(options) {
     'maps-me': 'mapsme://',
     osmand: isIOS ? 'osmandmaps://' : 'osmand.geo://',
     gett: 'gett://',
+    navermap:'nmap://'
```

Comment:
Only allow Naver to be detected when option is set:

```suggestion
    navermap: options.naverCallerName ? 'nmap://' : 'nmap-disabled://'
```